xarray_sentinel/esa_safe.py

```diff
@@ -21,6 +22,39 @@
 }
 
 
+def get_annotation_path(
+    product_path: T.Union[str, "os.PathLike[str]"],
+    subswath: str,
+    polarization: str = "vv",
+) -> "os.PathLike[str]":
+    manifest = open_manifest(product_path)
+    product_attrs, product_files = parse_manifest_sentinel1(manifest)
+
+    product_path = pathlib.Path(product_path)
+    if product_path.is_dir():
+        folder = product_path
+    else:
+        folder = product_path.parent
+
+    annotation_path = None
+    for file in product_files:
+        name = os.path.basename(file)
+        if re.match(
```

Comment:
I see how `re` is more precise, but `f"{subswath.lower()}-slc-{polarization.lower()}" in name` avoids an import. I would prefer it. @aurghs your call.

Reply:
I'd like to avoid the import of `re` and I have already tried what you suggest. Unfortunately with this you match also all these files:
- s1b-iw3-slc-vv-20210401t052623-20210401t052648-026269-032297-006.xml
- noise-s1b-iw3-slc-vv-20210401t052623-20210401t052648-026269-032297-006.xml
- calibration-s1b-iw3-slc-vv-20210401t052623-20210401t052648-026269-032297-006.xml
- s1b-iw3-slc-vv-20210401t052623-20210401t052648-026269-032297-006.tiff

And I prefer to use `re`, instead to use a double check.